README.md

```diff
@@ -136,3 +136,31 @@ elif ((current_hour >= 17 && current_hour < 24)) || ((current_hour >= 0 && curre
 fi
 ```
 
+## Main Quest: Moving from a JavaScript file to TypeScript
+
+our first step here is using a package manager(in our case Yarn) to install:
+```
+yarn add typescript
+```
+
+now we're going to rename our .js file to .ts using the following command:
+
+```
+rm program_name.js program_name.ts
+```
+In the program we just renamed above we'll have to replace:
+``` const express = require('express') ``` for ```import express from 'express';```
+
+Lastly we're going to run this sequence of command to finalize the setup:
+```
+yarn add @types/express
+yarn add ts-node
```

Comment:
Thoughts on having people run a single `yarn add` command that includes all the packages at once in the beginning?

Reply:
I thought about but maybe it's more beginner friendly if we separate them?